viewing_party/party.py

```diff
@@ -1,23 +1,125 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    if title and genre and rating:
+        movie_dict = {"title": title,
+        "genre": genre,
+        "rating": rating
+        }
+        return movie_dict
+    else:
+        return None
+
+def add_to_watched(user_data, movie):
+    watched_list = []
+    watched_list.append(movie)
+    user_data["watched"] = watched_list

```

Comment:
This has a bug that our tests don't catch.  If the `user_data["watched"]` list has content, this function will overwrite the list with a list containing only the new movie.
```suggestion
    user_data["watched"].append(movie)
```